src/libUtils/JsonUtils.h

```diff
@@ -51,14 +51,20 @@ class JSONUtils {
 
   /// Convert a string to Json object
   bool convertStrtoJson(const std::string& str, Json::Value& dstObj) {
-    std::string errors;
     std::lock_guard<std::mutex> g(m_mutexReader);
-    if (!m_reader->parse(str.c_str(), str.c_str() + str.size(), &dstObj,
-                         &errors)) {
-      LOG_GENERAL(WARNING, "Corrupted JSON: " << errors);
-      return false;
+    bool result = true;
+    try {
+      std::string errors;
+      if (!m_reader->parse(str.c_str(), str.c_str() + str.size(), &dstObj,
```

Comment:
Can put std::lock_guard<std::mutex> g(m_mutexReader); on top of m_reader->parse....?
So can easily find it is protected by mutex

Reply:
Done